kolibri/plugins/learn/assets/src/views/content-cards/content-carousel.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+
+  <section class="content-carousel">
+
+    <header v-if="header" class="content-carousel-header">
+      <h2 v-if="header"> {{header}} </h2>
+      <sub v-if="subheader"> {{subheader}} </sub>
```

Comment:
https://www.w3.org/TR/html5/text-level-semantics.html#the-sub-and-sup-elements

> The sup element represents a superscript and the sub element represents a subscript.

The subheader isn't subscript.